web/template/payments.gohtml

```diff
@@ -15,9 +15,45 @@
             {{ else }}
                 <table class="govuk-table my-table table__no-border">
                     <tbody class="govuk-table__body">
+                    <tr class="govuk-table__row">
+                        {{ if gt .OutstandingFee 0 }}
+                            <th scope="row" class="govuk-table__header">
+                                <h2 class="govuk-heading-m govuk-!-padding-top-3 govuk-!-padding-bottom-2">Outstanding fee due: </h2>
+                            </th>
```

Comment:
These aren't technically headers, we're just styling them to look like headers, so better to just apply the styles to the table cell (I've just copy-pasted but I suspect this might need a bit of fiddling, particularly with the padding):
```suggestion
                            <th scope="row" class="govuk-table__header govuk-heading-m govuk-!-padding-top-3 govuk-!-padding-bottom-2">
                                Outstanding fee due:
                            </th>
```